fractions/microprofile/microprofile-fault-tolerance/README.adoc

```diff
@@ -1,6 +1,5 @@
-
-= Microprofile Fault Tolerance Fraction
-:icons: font
+[[_microprofile_fault_tolerance]]
+= MicroProfile Fault Tolerance
 
 This fraction implements the https://github.com/eclipse/microprofile-fault-tolerance[Eclipse Microprofile Fault tolerance API^].
```

Comment:
Microprofile -> MicroProfile

Fault tolerance -> Fault Tolerance

Reply:
Ok, I'll add this change. You know I did not review the whole readme, just changed the title ;-)